packages/core/src/actions/public/parse-create-account-execute-result.ts

```diff
@@ -0,0 +1,26 @@
+import { ExecuteResult } from '@cosmjs/cosmwasm-stargate'
+import { ACCOUNT_ID_CHAIN_DELIMITER, AccountId } from '../../utils/account-id'
+import { chainIdToName } from '../../utils/chain-registry'
+import { findAbstractAttribute } from '../../utils/events'
+
+export function parseCreateAccountExecuteResult(
```

Comment:
Nice, this is going to be pretty helpful already.

Reply:
Although the createAccount method is available in the AccountFactoryClient itself I will provide the same alias functions that have been declared in the AbstractAccountClient class.